src/test/java/uk/gov/pay/connector/paymentprocessor/service/CardCaptureServiceTest.java

```diff
@@ -154,7 +154,14 @@ private void worldpayWillRespondWithSuccess() {
 
     private void stripeWillRespondWithSuccess() {
         when(mockedPaymentProvider.capture(any())).thenReturn(
-                fromBaseCaptureResponse(BaseCaptureResponse.fromTransactionId(randomUUID().toString(), STRIPE), PENDING, 50L, List.of(Fee.of(FeeType.TRANSACTION, 50L))));
+                fromBaseCaptureResponse(BaseCaptureResponse.fromTransactionId(randomUUID().toString(), STRIPE), PENDING, 50L,
+                        List.of(Fee.of(FeeType.TRANSACTION, 50L))));
+    }
+
+    private void stripeWillRespondWithSuccessV2() {
```

Comment:
could possibly name it differently to be clear

```suggestion
    private void stripeWillRespondWithSuccessAndAdditionalFees() {
```